srml/treasury/src/lib.rs

```diff
@@ -431,11 +433,11 @@ mod tests {
 	type Treasury = Module<Test>;
 
 	fn new_test_ext() -> runtime_io::TestExternalities<Blake2Hasher> {
-		let mut t = system::GenesisConfig::default().build_storage::<Test>().unwrap().0;
-		t.extend(balances::GenesisConfig::<Test>{
+		let mut t = system::GenesisConfig::default().build_storage::<Test>().unwrap();
+		extend_storage_overlays(&mut t, balances::GenesisConfig::<Test>{
 			balances: vec![(0, 100), (1, 99), (2, 1)],
 			vesting: vec![],
-		}.build_storage().unwrap().0);
+		}.build_storage().unwrap());
```

Comment:
ok this can be done this way but I think it is preferable if we actually always do as the genesis config is build which means module storage are assimilated in order.

Thus I would prefer that on test even if it is not needed ppl use this method so first is build and other are assimilated (or all are assimilated from empty ones)

Reply:
I am not 100% sure about order for my assimilates (I think I reversed initial order here but it seems logic to me that system is loaded before balance)